lib/libvcc/vcc_compile.c

```diff
@@ -447,6 +451,55 @@ vcc_destroy_source(struct source *sp)
 	free(sp);
 }
 
+/*--------------------------------------------------------------------
+ * Include files relative to source's path or vcl_dir
+ *
+ */
+
+static struct vsb *vcc_include_path(const struct vcc *tl, const char *fn) {
```

Comment:
Please keep the style of the file, i.e. return type goes in the previous line and brace goes in the following line.